frontend/src/state/index.ts

```diff
@@ -208,4 +208,20 @@ export const getIsAdvisorFromState = (state: AppState) => {
  * SAFE GET FUNCTIONS
  * Should only need to use these where there is not guaranteed to have a user
  */
-export const safelyGetIsAdvisorFromState = (state: AppState) => state.userState.user?.isAdvisor;
\ No newline at end of file
+
+export const safelyGetActivePlanFromState = (state: AppState) => {
```

Comment:
wanted to keep at least this function and safelyGetActivePlanScheduleFromState as we use the add class modal in the onboarding flow. I think we can remove the other ones but also no harm in keeping it. Thoughts?

Reply:
I think it's best to keep them. We only want to use the safe ones if we have to.